### PR TITLE
Nfr02 styling

### PR DESCRIPTION
**Pull request by Shandorin**

### CODE REVIEW

File: front/src/navbar/Nav.js [hunk 1]
```diff
@@ -191,9 +189,13 @@ const SideMenu = ({setName, superuser}) => {
       {openDial ? <DialogWindow /> : "" }
       
     </div>
+
```
Comment (Liljenmaa) [inline front/src/navbar/Nav.js:192]: Remove whitespace lines

File: front/src/weekview/Weekview.css [hunk 2]
```diff
@@ -172,139 +195,84 @@
...
+   display: inline-block;
+}
+
+.relative-text {
+   position: absolute;
+   bottom: 0;
+   left: 45px;
+}
+
+/*#weekDay {
```
Comment (Liljenmaa) [inline front/src/weekview/Weekview.css:223]: Prefer not to leave comment artifacts

Comment (Shandorin) [reply front/src/weekview/Weekview.css]: Fixed with newest updates

File: front/src/navbar/Nav.js (hunk 1 above)
Comment (Shandorin) [reply front/src/navbar/Nav.js]: Fixed with newest updates

**Review by TuoLuo994 — APPROVED** [review]
Great!

**Pull request merged**